src/utils/mui/style.ts

```diff
@@ -0,0 +1,14 @@
+export const flex = {
+	display: 'flex',
+	justifyContent: 'center',
+	alignItems: 'center',
+};
+
+export const circle = {
```

Comment:
what doesn mean circle? for what?

Reply:
This circle is before the text in the leaderboard title